src/Components/Blazor/WebAssembly.Authentication/src/Services/IAccessTokenProvider.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// Represents a contract for services capable of provisioning access tokens for an application.
+    /// </summary>
+    public interface IAccessTokenProvider
+    {
+        /// <summary>
+        /// Tries to get an access token for the current user with the default set of permissions.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Nit: Missing `<returns>` XML doc content here and below.

Reply:
done